notebooks/Project and Developer Similarity.ipynb

```diff
@@ -0,0 +1,625 @@
+{
```

Comment:
<p>Same suggestion/question on Pandas, e.g regarding the meaning of the <code>topics_identifiers_df.iloc[1:, :].values.argsort()[:, -10:][:, ::-1]</code> that might not be immediately obvious for developers with non-DS background.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/mloncode/devfest2019-workshop/pull/6/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='27'/>